src/libs/ReportUtils.ts

```diff
@@ -3948,41 +3692,34 @@ function getTaskAssigneeChatOnyxData(accountID, assigneeEmail, assigneeAccountID
 
 /**
  * Returns an array of the participants Ids of a report
- *
- * @param {Object} report
- * @returns {Array}
  */
-function getParticipantsIDs(report) {
+function getParticipantsIDs(report: OnyxEntry<Report>): Array<number | undefined> {
```

Comment:
```suggestion
function getParticipantsIDs(report: OnyxEntry<Report>): number[] {
```

If you are filtering undefined values, I believe it should be only `number`.